code/__HELPERS/roundend.dm

```diff
@@ -815,9 +815,10 @@ GLOBAL_VAR(survivor_report) //! Contains shared survivor report for roundend rep
 	discordmsg += "Server: [CONFIG_GET(string/servername)]\n"
 	discordmsg += "Round Number: [GLOB.round_id]\n"
 	discordmsg += "Duration: [DisplayTimeText(world.time - SSticker.round_start_time)]\n"
-	discordmsg += "Players: [GLOB.player_list.len]\n"
+	discordmsg += "Players: [GLOB.player_list.len] (Total: [GLOB.connected_ckeys.len])\n"
```

Comment:
```suggestion
	discordmsg += "Players: [GLOB.player_list.len] (Current connections: [GLOB.connected_ckeys.len])\n"
``` 'Total' sounds it may bring misinterpretation...

Reply:
It's the total number of connections throughout the entire round, I'll change the name of `GLOB.connected_ckeys`